src/tree/updater_gpu_hist.cu

```diff
@@ -251,15 +251,15 @@ struct DeviceHistogram {
   thrust::device_vector<GradientPairSumT::ValueT> data;
   const size_t kStopGrowingSize = 1 << 26;  // Do not grow beyond this size
   int n_bins;
-  int device_idx;
+  int normalised_device_idx;
```

Comment:
This should all be device_idx, as before let's remove all normalised/unnormalised terminology.